src/js/bsp-modal.js

```diff
@@ -189,6 +171,24 @@ var bsp_modal = {
         var self = this;
 
         vex.close(self.vexInstance.data().vex.id);
+    },
+
+    // public API that recenters the modal in the window screen (useful after content changes)
+    // alternatively if the modal is bigger than the window, we have to CSS hack
+    // the height of the overlay to cover the whole modal
+    recenter: function() {
```

Comment:
This function was relocated from the private function so devs have access to it. Minor improvements were made.